fontend/app/products/[productId]/page.tsx

```
@@ -578,9 +1345,22 @@
                             <div>
-                                <h2 className="text-3xl font-bold text-red-600">
-                                    {selectedVariant ? formatPrice(selectedVariant.price) : (product.variants.length > 0 && product.variants[0].price ? formatPrice(product.variants[0].price) : 'N/A')}
-                                </h2>
+                                {selectedVariant && selectedVariant.salePrice && selectedVariant.salePrice < selectedVariant.price ? (
+                                    // Trường hợp CÓ GIÁ SALE
+                                    <div className="flex items-baseline gap-3">
+                                        <span className="text-2xl text-gray-500 line-through">
+                                            {formatPrice(selectedVariant.price)}
+                                        </span>
+                                        <span className="text-3xl font-bold text-red-600">
+                                            {formatPrice(selectedVariant.salePrice)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)